data/apply-ne_country_label_recasting.sql

```diff
@@ -0,0 +1,38 @@
+-- Recasts a few rows in the ne_10m_admin_0_countries_iso and ne_10m_admin_0_countries_tlc
+-- tables to either show or hide certain labels
+
+-- sets featurcla to equal fclass_tlc values. We can then further modify the values while keeping the fclass intact.
```

Comment:
NIt: It'd be helpful to know that this also yields **Kosovo** and **Taiwan** as `country` deliberately. Please add that as a comment here.

Reply:
-- The TLC table yields around 10 country and dependency features over ISO (including Kosovo and Taiwan), which Tilezen sometimes has opinions about.